src/components/Dropdown/Dropdown.st.css

```diff
@@ -116,13 +253,17 @@
 }
 
 .dropdown::popoverContent {
-    border-width: 0;
+    border-width: 1px;
+    border-color: rgba(0, 0, 0, 0.4);
     box-sizing: border-box;
     /* Popover fix. For some reason popover has inline style with transform: translate3d(-1px, 42px, 0px); */
     margin-left: 1px;
     max-width: 100%;
     box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1), 0 0 4px 0 rgba(0, 0, 0, 0.1);
-    background-color: "color(color-1)";
```

Comment:
Sorry for the ping-pong review 🙈  but I just noticed 2 things:
1. I think the box-shadow defined above is the border you've added. In that case I think there's no need for the extra border.
2. I see you removed the `background-color` here, is there a reason? I think it should remain here for the default styling, no?

Reply:
will merge the box shadow & the border.
regarding the `background-color` - removed it per Johanna's request (to change the hover color)

Reply:
removing the background color might cause your teams apps to break 🙈 
We already had this issue a few weeks ago, where I changed the background color and it broke the product gallery page or something like that.

Also I see the background color can be overriden (a few lines below - 265), so why shouldn't we have a default background color?

Also I'm not sure I understand, when overriding we do set the

Reply:
The same goes the border color.
Also I think we should check with Johanna if the box shadow is necessary then.

Reply:
OK, so first, we need to bring back the `background-color`
I don't know why it's written like this (using quotation marks).
It can just be: 
```css
background-color: color(color-1);
``` 

Also, we need to add the border, which isn't overridable.

It should probably be something like this:
```css
border: 1px solid applyOpacity(value(CurrentItemTextColor), 0.4);
```